src/.editorconfig

```diff
@@ -159,6 +159,8 @@ csharp_style_prefer_pattern_matching = true:silent
 csharp_style_prefer_not_pattern = true:suggestion
 csharp_style_prefer_extended_property_pattern = true:suggestion
 csharp_indent_case_contents_when_block = false
+csharp_style_prefer_top_level_statements = true:silent
+csharp_style_prefer_primary_constructors = true:suggestion
```

Comment:
these changes were made automatically by VS, i don't know exactly what they do, can remove it i suppose